test/data/source/section_2_sentinel.tif.aux.xml

```diff
@@ -0,0 +1,72 @@
+<PAMDataset>
```

Comment:
Did you want to include these xml files or are they here accidentally?

Reply:
Fixed with 34abe54